tests/python/pants_test/backend/python/tasks/BUILD

```diff
@@ -19,94 +18,46 @@ python_library(
 )
 
 python_tests(
-  name='python_task',
-  sources=['test_python_task.py'],
-  dependencies=[
-    ':python_task_test_base',
-    'src/python/pants/backend/python/tasks:python',
-  ]
-)
-
-python_tests(
-  name='python_binary_create',
-  sources=['test_python_binary_create.py'],
+  sources=globs('test_*.py', exclude=[globs('*_integration.py')]),
   dependencies=[
     ':python_task_test_base',
-    'src/python/pants/backend/python/tasks:python',
-    'src/python/pants/base:run_info',
-  ]
-)
-
-python_tests(
-  name='pytest_run',
-  sources=['test_pytest_run.py'],
-  dependencies=[
-    '3rdparty/python:pex',
+    '3rdparty/python/twitter/commons:twitter.common.collections',
+    '3rdparty/python/twitter/commons:twitter.common.dirutil',
+    '3rdparty/python:coverage',
     '3rdparty/python:mock',
-    ':python_task_test_base',
-    'src/python/pants/backend/python/tasks:python',
-    'src/python/pants/util:contextutil',
-  ]
-)
-
-python_tests(
-  name='python_repl',
-  sources=['test_python_repl.py'],
-  dependencies=[
-    ':python_task_test_base',
-        'src/python/pants/backend/python/tasks:python',
-    'src/python/pants/backend/python:all_utils',
+    '3rdparty/python:pex',
+    'src/python/pants/backend/python/subsystems',
+    'src/python/pants/backend/python/targets',
+    'src/python/pants/backend/python/tasks',
+    'src/python/pants/backend/python:interpreter_cache',
+    'src/python/pants/backend/python:python_artifact',
+    'src/python/pants/backend/python:python_requirement',
+    'src/python/pants/base:build_root',
     'src/python/pants/base:exceptions',
+    'src/python/pants/base:run_info',
     'src/python/pants/build_graph',
-    'src/python/pants/task',
+    'src/python/pants/fs',
+    'src/python/pants/python',
+    'src/python/pants/source',
     'src/python/pants/util:contextutil',
-  ]
-)
-
-python_tests(
```

Comment:
So it looks like the python isort tests were nuked when they should have remained (they had stayed in the old tasks dir instead of being in tasks2, because they weren't affected by the new pipeline changes).